utils/viz_utils.py

```diff
@@ -82,8 +82,8 @@ def __init__(
             "task_info",
             "target_position",
         ),
-        x: str = "x",
-        y: str = "z",
+        path_to_x: Sequence[str] = ("x",),
+        path_to_y: Sequence[str] = ("z",),
```

Comment:
Just double checking, that it is `path_to_y` and `("z",)` seems surprising. I assume this is just because unity uses (x,z) as the floor plane coordinates and `y` for vertical translations?

Reply:
That's correct.